docs/releases/1.11.txt

```diff
@@ -150,7 +150,8 @@ CSRF
 Database backends
 ~~~~~~~~~~~~~~~~~
 
-* ...
+* Added support for ``SKIP LOCKED`` to :meth:`QuerySet.select_for_update()
```

Comment:
"Added the skip_locked argument to...."

For less verbosity, you can write the link as:
```
:meth:`.QuerySet.select_for_update()`
````